common.lic

```diff
@@ -93,6 +93,16 @@ module DRC
     end
     true
   end
+  
+  def collect(item)
+    case bput("collect #{item}",'You manage to collect a pile', 'The room is too cluttered')
+    when 'The room is too cluttered'
+      fput('kick pile')
+      fput('kick pile')
+      fput('kick pile')
+      fput("collect #{item}")
+    end
```

Comment:
I recommend `waitrt?` at the end of the method (so you don't have to do it every time you call it).